expanse.service

```
@@ -0,0 +1,16 @@
+[Unit]
+Description=web app for storing Reddit items
+Documentation=https://github.com/jc9108/expanse
+Wants=network-online.target
+Requires=docker.service
+After=network-online.target docker.service
+
+[Service]
+Type=exec
+WorkingDirectory=/opt/expanse
+ExecStartPre=-run.sh prod update
```

```suggestion
ExecStartPre=-sh ./run.sh prod update
```
add `sh`, use relative path

---

The included shebang will set the interpreter used to execute the script, and I seem to recall that as the preferred method for using scripts in systemd rather than overriding the included shebang, but I could be wrong about that (I'm still at dinner right now). Just keep in mind if you or any fork decides to change interpreter both the script and unit file would need updating.

---

if thats the preferred way for systemd then go ahead and remove the `sh`. please keep the relative paths though